main.ts

```diff
@@ -0,0 +1,100 @@
+import puppeteer from "./deps/puppeteer.ts"
+import { parse } from "./deps/std/flags.ts"
+import * as path from "./deps/std/path.ts"
+import { run, runWithBrowser, runWithDeno } from "./run.ts"
+
+const flags = parse(Deno.args, {
+  string: [
+    "browser-exec-path",
+    "concurrency",
+    "dir",
+    "filter",
+    "ignore",
+    "import-map",
+    "output",
+    "reload-url",
+  ],
+  boolean: ["browser", "headless"],
+  default: {
+    "browser-exec-path": "/usr/bin/chromium",
+    headless: true,
+    ignore: ".ignore",
+    "reload-url": "http://localhost:4646",
```

Comment:
The default for this shouldn't be capi-specific. I think ideally this should be named `--reload`, with an alias of `-r`, to align with Deno.